packages/block-editor/src/components/inserter/tabs.js

```diff
@@ -39,9 +39,9 @@ function InserterTabs( { onSelect, children, onClose, selectedTab }, ref ) {
 	return (
 		<div className="block-editor-inserter__tabs" ref={ ref }>
 			<Tabs onSelect={ onSelect } selectedTabId={ selectedTab }>
-				<div className="block-editor-inserter-sidebar__header">
+				<div className="block-editor-inserter__tablist-and-close">
```

Comment:
Minor one. I'm not the best person to suggest a name, but it sounds a little weird at first glance. What's wrong with `block-editor-inserter-sidebar__header`? Maybe `block-editor-inserter__tablist-buttons`?
Feel free to ignore it.